openedx/core/djangoapps/user_api/preferences/views.py

```diff
@@ -263,3 +270,50 @@ def delete(self, request, username, preference_key):
             return Response(status=status.HTTP_404_NOT_FOUND)
 
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class ChangeToVerifiedMode(APIView):
+    """
+    Change the status to 'approved' in Software Secure Photo Verification model for the given user.
+    """
+    authentication_classes = (JwtAuthentication,)
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request, username):
```

Comment:
What do you think about using a post request ratter? In fact we are not getting a resource, we are creating or modifying a resource.

Reply:
Being a post request, we can send the username on the request payload

Reply:
Ok, we'd that on mind.